client/request/commit.go

```diff
@@ -19,10 +19,10 @@ var (
 type CommitSelect struct {
 	Field
 
-	DocKey    immutable.Option[string]
-	FieldName immutable.Option[string]
-	Cid       immutable.Option[string]
-	Depth     immutable.Option[uint64]
+	DocKey      immutable.Option[string]
+	FieldIDName immutable.Option[string]
```

Comment:
thought: I'm not sure `FieldIDName` makes sense here. `FiedID` seemed more appropriate.